src/components/DataViewFilterMultiselect/DataViewFilterMultiselectTypes.tsx

```diff
@@ -0,0 +1,40 @@
+type ValueTypes = string[];
```

Comment:
All of this file has already been worked in another branch, so I'd recommend commenting it for now so it doesn't conflict when merging